### ISSUE TITLE
Find case examples of large accessible hideable navigation

### ISSUE THREAD

**Issue by bengolder** [rootandrebound/roadmap-to-html#31]
Criteria:

- should be easily navigable using voiceover or keyboard
- should be able to "slide" away the navigation to hide it
- should work on mobile

Examples:
- [Teach Access Tutorial Table of Contents](https://teachaccess.github.io/tutorial/)
